2/contrib/s2i/run

```diff
@@ -152,6 +150,10 @@ if [ ! -z "${CONTAINER_CORE_LIMIT}" ]; then
     JAVA_CORE_LIMIT="-XX:ParallelGCThreads=${CONTAINER_CORE_LIMIT} -Djava.util.concurrent.ForkJoinPool.common.parallelism=${CONTAINER_CORE_LIMT} -XX:CICompilerCount=2"
 fi
 
+if [ -z "${JVM_ARGS}" ]; then
+    JVM_ARGS="-XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap -Dsun.zip.disableMemoryMapping=true"
+fi
```

Comment:
Looks like you could reuse the JAVA_OPTS env var we already support for this (today it looks like JAVA_OPTS doesn't get set by us, only by the user, so you could make this the default value for it now).

Reply:
Right. But what if existing users have other JAVA_OPTS? If they don't have those options part of their JAVA_OPTS now they would end up with a container java that's not looking at cgroup settings. This new variable would ensure people would get this always. Thoughts?

Reply:
yeah i considered that.  my feeling is most people probably don't have JAVA_OPTS today and I really don't want to end up supporting more env variables in this image than we already do (there are already too many and they're not nicely organized, unfortunately).